src/Send/AssetSelectorScreen/AssetSelectorScreen.tsx

```diff
@@ -42,7 +42,7 @@ export const AssetSelectorScreen = ({balances}: Props) => {
 
   const sortedBalance: Array<[TokenId, Quantity]> = Object.entries(balances)
     .sort(([, quantityA]: [TokenId, Quantity], [, quantityB]: [TokenId, Quantity]) =>
-      Quantities.isGreaterThan(quantityA, quantityB) === true ? -1 : 1,
+      parseInt(Quantities.diff(quantityB, quantityA), 10),
```

Comment:
what does this change do?

Reply:
@stackchain suggestion

Reply:
yap this was @wolverineks first suggestion when we didn't have `isGreaterThan`.